api/v1alpha1/application.go

```diff
@@ -89,12 +89,37 @@ type ApplicationSpec struct {
 
 	//+kubebuilder:validation:Optional
 	ResourceLabels map[string]map[string]string `json:"resourceLabels,omitempty"`
+
+	//+kubebuilder:validation:Optional
+	AuthorizationSettings *AuthorizationSettings `json:"authorizationSettings,omitempty"`
+}
+
+// AuthorizationSettings Settings for overriding the default deny of all actuator endpoints. AllowAll will allow any
+// endpoint to be exposed. Use AllowList to only allow specific endpoints.
+//
+// Please be aware that actuator endpoints may expose information about your application which you do not want to expose.
+// Before allow listing actuator endpoints, please be aware of what these endpoints will expose, especially if your application is served via an external ingress.
```

Comment:
Remove references to actuator

Reply:
Should it not mention somewhere that, by default, all actuator endpoints are denied? I agree with using HTTP instead of actuator in these cases though. Just wondering if it should be an added note in the docs :)

Reply:
I thought about it wrong, nevermind me ¯\_(ツ)_/¯